Makefile

```diff
@@ -11,7 +11,7 @@ setup: ## Generate ssl certificate for development.
 	cd sslcert; openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout tls.key -out tls.crt -config req.conf -extensions 'v3_req'
 
 run: ## Run the service locally.
-	go run -tags development main.go -v=9
+	go run -tags development main.go -v=3
```

Comment:
Lowered to `v=3` because the leader election lib starts polluting the log at `v=4`